Jenkinsfile-utils.groovy

```diff
@@ -0,0 +1,109 @@
+/*
```

Comment:
Does this file have to have a `.groovy` extension? Maybe we can name the file `Jenkinsfile-utils` and then add this to the first line of the file:

```
#!/usr/bin/env groovy
```